tests/core_test/containers_test/archive_test.py

```diff
@@ -0,0 +1,177 @@
+import jax.numpy as jnp
+import pytest
+
+from qdax.core.containers.archive import Archive, score_euclidean_novelty
+
+a = jnp.array([[0.0, 0.0], [0.0, 1.0]])
+b = jnp.array([[1.0, 0.0], [1.0, 1.0]])
+c = jnp.array([[2.0, 0.0], [2.0, 1.0]])
+d = jnp.array([[0.0, 2.0], [1.0, 2.0]])
+e = jnp.array([[2.0, 2.0], [3.0, 2.0]])
+f = jnp.array([[0.1, 0.0], [0.9, 0.0]])
+
+fake_border = jnp.nan
+
+# TODO: missing cases
```

Comment:
Do we want to resolve this TODO or let it in the code?

Reply:
I removed it! Thanks!